src/reference/00-Getting-Started/03-Directories.md

```diff
@@ -50,6 +46,10 @@ src/
 Other directories in `src/` will be ignored. Additionally, all hidden
 directories will be ignored.
 
+Source code can be placed in the project's base directory as
+`hello/app.scala`. However, most people don't do this for real projects;
+too much clutter.
```

Comment:
I think we have to for the sake of completeness. This is a page titled Directories, and it talks about the places where sbt will look for `*.scala` file out of the box. Since sbt is coded to look for them the base directory, we should mention it.

Another reason to mention this is because it does become relevant/recommended for the meta-build. See http://www.scala-sbt.org/0.13/docs/Organizing-Build.html#sbt+is+recursive